Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,64 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+            /* Splitting the equation into three parts A,B,C */
+
+            string[] token = equation.Split("=");

```

Comment:
use 'var' rather than specific type for variable declarations wherever possible.
Using 'var' makes your code more readable, using type names makes code little messy when you have large type names, so make it habit of using 'var' for variable declarations whenever possible


See if you can List<string> rather than string[]